test/test/jfr/Ttsp.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.jfr;
+
+import java.util.Random;
+
+class Ttsp {
+
+    static private byte loop() {
+        byte[] byteArray = new byte[1024 * 1024 * 1024];
+        for (int i = 0; i < 10000; i++) {
+            new Random().nextBytes(byteArray);
```

Comment:
`Random.nextBytes` contains a loop with a safepoint check - it does not cause long time-to-safepoint pauses.